src/node-lib/curriculum-api-2023/shared.schema.new.ts

```diff
@@ -0,0 +1,263 @@
+import { z } from "zod";
```

Comment:
These are the source schemas but they are immediately derived for parsing against the gql query and then later converted to camel case.